src/patterns/components/footer/optional.hbs

```diff
@@ -1,7 +1,8 @@
 ---
 name: Base With Optional Elements
 description: |
-  The base footer can be used with all or some of the optional elements. The optional items are feedback links,
+  The base Footer can be used with all or some of the optional elements.
+  The optional items are feedback links,
   social links and three columns of site links.
```

Comment:
The comma styles are inconsistent - is there a copy style guide to follow? Oxford comma, no oxford comma? etc.

Reply:
@scottux we were planning on having content review